src/app/pages/learning-research/learning-research.js

```diff
@@ -0,0 +1,46 @@
+import LoaderPage from '~/components/jsx-helpers/loader-page';
+import React from 'react';
+import LazyLoad from 'react-lazyload';
+import '~/pages/learning-research/research.scss';
+import {Banner, Header} from '~/pages/learning-research/components/header';
+import {ResearchSection} from '~/pages/learning-research/components/research-areas';
+import {Funders} from '~/pages/learning-research/components/funders';
+import {ContactUs} from '~/pages/learning-research/components/contact-us';
+import {Publications} from '~/pages/learning-research/components/publications';
+import {MembersSection} from '~/pages/learning-research/components/members';
+
+function LearningResearchPage({data}) {
+    return (
+        <React.Fragment>
+            <LazyLoad>
```

Comment:
Generally speaking, there's no reason to lazy load the first element of a page. Only the elements that are not in the viewport when the page loads benefit.